ci-operator/step-registry/openshift/claude/payload/agent/openshift-claude-payload-agent-commands.sh

```
@@ -215,10 +215,43 @@ tar -czf "${ARTIFACT_DIR}/snapshot-${PAYLOAD_TAG}.tar.gz" -C "${SNAPSHOT_DIR}" .
 SNAPSHOT_DATA_DIR=$(dirname "$(find "${SNAPSHOT_DIR}" -name summary.json -print -quit)")
 echo "Snapshot data dir: ${SNAPSHOT_DATA_DIR}"
 
-# Install the must-gather plugin for analyzing must-gather archives
-echo "Installing must-gather plugin..."
+# Use a temporary config dir so we install plugins from the test fork
+export CLAUDE_CONFIG_DIR=$(mktemp -d)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)